### PR TITLE
Service IPAllocator cleanups

### PR DESCRIPTION
**Pull request by danwinship**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Spinoff of #90115. It is not clear if that will ever merge, but the cleanups there still make sense on their own

**Which issue(s) this PR fixes**:
none

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

/sig network
/priority backlog

### CODE REVIEW

File: pkg/registry/core/service/ipallocator/allocator_test.go [hunk 1]
```diff
@@ -30,29 +30,33 @@ func TestAllocate(t *testing.T) {
...
 	}{
 		{
 			name:             "IPv4",
 			cidr:             "192.168.1.0/24",
 			free:             254,
 			released:         "192.168.1.5",
-			outOfRange1:      "192.168.0.1",
-			outOfRange2:      "192.168.1.0",
-			outOfRange3:      "192.168.1.255",
+			outOfRange:       []string{
```
Comment (aojea) [inline pkg/registry/core/service/ipallocator/allocator_test.go:41]: gofmt is complaining here 

```
Run ./hack/update-gofmt.sh
```

**Review by thockin — COMMENTED** [review]
Thanks!

/lgtm
/approve

**Pull request merged**